ft_freqanalysis.m

```diff
@@ -106,10 +106,18 @@
 %   cfg.foilim     = [begin end], frequency band of interest
 %   cfg.toi        = vector 1 x numtoi, the times on which the analysis
 %                    windows should be centered (in seconds)
-%   cfg.width      = 'width', or number of cycles, of the wavelet (default = 7)
+%   cfg.width      = 'width', or number of cycles, of the wavelet or base wavelet for 
+%                    superlets(default = 7)
 %   cfg.gwidth     = determines the length of the used wavelets in standard
 %                    deviations of the implicit Gaussian kernel and should
 %                    be choosen >= 3; (default = 3)
+%   cfg.superlets  = 'no', 'additive', 'multiplicative' (default = 'no')
+%                    If set to value other than 'no' performs superlet spectral estimation 
+%                    according to Moca et al. 2019 (https://doi.org/10.1101/583732), i.e.
```

Comment:
Hmmm, so this is so far published only as a preprint? Do you know whether it is accepted as a peer-reviewed paper somewhere?

Reply:
Yes, it's only a preprint so far. According to the senior author it's been submitted to some journal.
I'm not affiliated with the authors in any way, so I can't really say more about this.